MMM-Spotify.js

```diff
@@ -525,13 +636,84 @@ Module.register("MMM-Spotify", {
     return progress;
   },
 
+  getCoverContainer: function () {
+    const cover_img = this.getHTMLElementWithID('img', "SPOTIFY_COVER_IMAGE")
+    cover_img.className = 'fade-in'
+    cover_img.src = "./modules/MMM-Spotify/resources/spotify-xxl.png"
+
+    const cover = this.getHTMLElementWithID('div', "SPOTIFY_COVER")
+    cover.appendChild(cover_img)
+    return cover
+  },
+
+  getMinimalistBarDom: function (container) {
+    container.appendChild(this.getProgressContainer())
+
+    const misc = this.getHTMLElementWithID('div', "SPOTIFY_MISC")
+    misc.appendChild(this.getHTMLElementWithID('div', "SPOTIFY_BACKGROUND"))
+    misc.appendChild(this.getDeviceContainer())
+
+    const info = this.getHTMLElementWithID('div', "SPOTIFY_INFO")
+    if (this.config.miniBarScroll) info.className = 'marquee';
+
+    const infoElements = [
+      "SPOTIFY_TITLE",
+      "SPOTIFY_ALBUM",
+      "SPOTIFY_ARTIST",
+    ]
+
+    infoElements.forEach(key => {
+      const element = this.getHTMLElementWithID('div', key)
+      element.appendChild(this.getEmptyTextHTMLElement())
+      info.appendChild(element)
+    })
```

Comment:
This should work better, no need to manipulate text in updatePlaying()

```
infoElements.forEach((key, index) => {
      if (index > 0) {
        const bulletElement = this.getHTMLElementWithID("span", "TEXT_BULLET");
        bulletElement.innerHTML = "&#8226;"
        info.appendChild(bulletElement)
      }
      const element = this.getHTMLElementWithID('div', key)
      element.appendChild(this.getEmptyTextHTMLElement())
      info.appendChild(element)
    })
```

Also add this to css

```
#SPOTIFY.minimalistBar #TEXT_BULLET {
  margin-right: 0.5em;
  white-space: nowrap;
  display:inline-block;
  color: grey;
}
```